### PR TITLE
Model/vit

### PR DESCRIPTION
**Pull request by adhiiisetiawan**

### CODE REVIEW

**Comment by rubentea16** [comment]
image
ngapain di freeze? @adhiiisetiawan

**Comment by rubentea16** [comment]
image
set default value buat labelsnya @adhiiisetiawan

**Comment by rubentea16** [comment]
kenapa nambahin Dropout?

**Comment by adhiiisetiawan** [comment]
> image ngapain di freeze? @adhiiisetiawan

kemarin pas nge train ku freeze, apa yang ini gaperlu di freeze?

**Comment by rubentea16** [comment]
> > image ngapain di freeze? @adhiiisetiawan
> 
> kemarin pas nge train ku freeze, apa yang ini gaperlu di freeze?

ga perlu di freeze

**Comment by adhiiisetiawan** [comment]
> > > image ngapain di freeze? @adhiiisetiawan
> > 
> > 
> > kemarin pas nge train ku freeze, apa yang ini gaperlu di freeze?
> 
> ga perlu di freeze

tapi pas bikin pretrained modelnya tetep pake freeze kah?

**Comment by rubentea16** [comment]
@adhiiisetiawan buka discord skrg
